src/common/helpers/hash-password.helpers.ts

```diff
@@ -0,0 +1,28 @@
+import crypto from 'crypto';
+import { promisify } from 'util';
+
+const randomBytes = promisify(crypto.randomBytes);
+const pbkdf2 = promisify(crypto.pbkdf2);
+
+const saltLength = 16;
+export const generateSalt = async (): Promise<string> => {
+  const salt = await randomBytes(saltLength);
+  return salt.toString('hex');
+};
+
+const passwordLength = 64;
+const digestAlgorithm = 'sha512';
+const iterations = 100000;
+export const hashPassword = async (
+  password: string,
+  salt: string
+): Promise<string> => {
+  const hashedPassword = await pbkdf2(
+    password,
+    salt,
+    iterations,
+    passwordLength,
+    digestAlgorithm
+  );
+  return hashedPassword.toString('hex');
+};
```

Comment:
let's use this new approach https://github.com/Lemoncode/bootcamp-backend/blob/main/00-stack-documental/04-rest-api/06-auth/03-hashing-passwords/README.md